stable/datadog/templates/deployment.yaml

```diff
@@ -1,25 +1,37 @@
+{{- if .Values.deployment.enabled }}
 {{- if .Values.datadog.apiKey }}
-{{- if .Values.datadog.collectEvents }}
 apiVersion: extensions/v1beta1
 kind: Deployment
 metadata:
-  name: {{ template "fullname" . }}-events
+  name: {{ template "fullname" . }}
   labels:
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
 spec:
-  replicas: 1
+  replicas: {{ .Values.deployment.replicas }}
   template:
     metadata:
       labels:
-        app: {{ template "fullname" . }}-events
-      name: {{ template "fullname" . }}-events
+        app: {{ template "fullname" . }}
+      name: {{ template "fullname" . }}
     spec:
       containers:
-      - name: {{ .Chart.Name }}-events
+      - name: {{ .Chart.Name }}
         image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
         imagePullPolicy: {{ .Values.image.pullPolicy }}
         resources:
 {{ toYaml .Values.resources | indent 12 }}
+        ports:
+        - containerPort: 8125
```

Comment:
Hey @ivanilves, I'm still trying to understand the use case for Deployments. When would an cluster operator opt to provide a per team/namespace datadog metric collection (instead of cluster wide)?